cmd/coder/urls.go

```diff
@@ -14,13 +15,124 @@ import (
 )
 
 type urlsCmd struct{}
+type createURLCmd struct{}
+type delURLCmd struct{}
 
+// DevURL is the parsed json response record for a devURL from cemanager
 type DevURL struct {
+	ID     string `json:"id"`
 	URL    string `json:"url"`
 	Port   string `json:"port"`
 	Access string `json:"access"`
 }
 
+var urlAccessLevel = map[string]string{
+	"PRIVATE": "Only you can access",
+	"ORG":     "All members of your organization can access",
+	"AUTHED":  "Authenticated users can access",
+	"PUBLIC":  "Anyone on the internet can access this link",
+}
+
+func isAccessLevelValid(level string) bool {
+	_, ok := urlAccessLevel[level]
+	return ok
+}
+
+func (cmd createURLCmd) Spec() cli.CommandSpec {
+	return cli.CommandSpec{
+		Name:  "create",
+		Usage: "<env name> <port> [<access>]",
+		Desc: `create/update a devurl for external access
+  <access> is one of [PRIVATE | ORG | AUTHED | PUBLIC]`,
+	}
+}
+
+// Run() creates or updates a devURL, specified by env ID and port
```

Comment:
You can omit the `()`. Doc comments should start with just the identifier.

Reply:
Ah, I always forget which way it is, thanks :)